playbooks/roles/streisand-mirror/vars/openvpn.yml

```diff
@@ -7,7 +7,7 @@ openvpn_mirror_href_base: "/mirror/openvpn"
 openvpn_samuli_seppanen_key_id: "0x29584D9F40864578"
 openvpn_samuli_seppanen_expected_fingerprint: "Key fingerprint = 6D04 F8F1 B017 3111 F499  795E 2958 4D9F 4086 4578"
 
-openvpn_base_download_url: "https://build.openvpn.net/downloads/releases/latest/"
```

Comment:
Correct, when it builds the link the difference is

`https://build.openvpn.net/downloads/releases//latest/openvpn-latest-stable.tar.gz` to `https://build.openvpn.net/downloads/releases/latest/openvpn-latest-stable.tar.gz`